compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -1151,7 +1151,7 @@ class OMR_EXTENSIBLE CodeGenerator
    void apply32BitLabelRelativeRelocation(int32_t * cursor, TR::LabelSymbol *);
    void apply32BitLabelTableRelocation(int32_t * cursor, TR::LabelSymbol *);
 
-
+   bool canEmitDataForExternallyRelocatableInstructions();
```

Comment:
Sorry, I didn't catch this before, but can we put this in the Power CodeGenerator rather than the base? This is specific to that CG so we don't really need it anywhere else right?

Reply:
Should be done in: 45fe12bd1